doc/command_ref.rst

```diff
@@ -979,15 +979,30 @@ Module subcommands take :ref:`\<module-spec>\ <specifying_modules-label>`... arg
     In case no profile was provided, all default profiles get installed.
     Module streams get enabled accordingly.
 
-    This command cannot be used for switching module streams. It is recommended to remove all
-    installed content from the module and reset the module using the
-    :ref:`reset <module_reset_command-label>` command. After you reset the module, you can install
-    the other stream.
+    This command cannot be used for switching module streams. It is recommended to use the
+    :ref:`dnf module switch <module_switch_command-label>` command.
```

Comment:
I think we can leave out the "It is recommended" part since we now have a proper solution and just go with:
> Use the :ref:`dnf module switch <module_switch_command-label>` command **for that**.